gr-uhd/grc/gen_uhd_usrp_blocks.py

```diff
@@ -202,7 +217,14 @@
     -   set_samp_rate(${'$'}{samp_rate})
     % for n in range(max_nchan):
     -   set_center_freq(${'$'}{${'center_freq' + str(n)}}, ${n})
+    % if sourk == 'source':
+    -   ${'$'}{'set_rx_agc(True, ${n})' if context.get('rx_agc${n}')() == 'Enabled' else ''}
```

Comment:
I wonder if we can't get those True/False values directly from the rx_agc var, but this code is touchy and anything that works is fine.